### PR TITLE
add support for action logits

### PR DESCRIPTION
**Pull request by yutaizhou**
certain algorithms like MADDPG's implementation sends logits to represent actions.

### CODE REVIEW

**Comment by yutaizhou** [comment]
@googlebot I signed it!

**Comment by qstanczyk** [comment]
Thank you for your pull request. Unfortunately we think it's not the right place to do this conversion. We try to follow Gym API where this is also not supported. To avoid confusion in cases when algorithm returns ndarray with a different semantics this conversion should be done on the algorithm side.
